js/binance.js

```diff
@@ -1003,8 +1010,11 @@ module.exports = class binance extends Exchange {
             };
             const status = this.safeString2 (market, 'status', 'contractStatus');
             const active = (status === 'TRADING');
-            const margin = this.safeValue (market, 'isMarginTradingAllowed', future || delivery);
-            const contractSize = this.safeFloat (market, 'contractSize');
+            const margin = this.safeValue (market, 'isMarginTradingAllowed', false);
+            let contractSize = undefined;
+            if (future || delivery) {
+                contractSize = this.safeFloat (market, 'contractSize', 1);
```

Comment:
Should we do safeNumber here?

Reply:
I'm using floats for the contractSize on purpose, since on line 3260 it is rounded